packages/appstore/src/components/containers/trading-app-card-actions.tsx

```diff
@@ -27,11 +27,12 @@ const TradingAppCardActions = ({
     is_account_being_created,
     is_buttons_disabled,
     is_real,
+    as_disabled,
 }: Actions) => {
     switch (action_type) {
         case 'get':
             return (
-                <Button disabled={is_account_being_created} primary_light onClick={() => onAction?.()}>
+                <Button disabled={is_account_being_created} primary_light onClick={onAction} as_disabled={as_disabled}>
```

Comment:
@sergei-deriv there is not `as_disabled` prop in the Button component 🤔 
 ```suggestion
                <Button disabled={is_account_being_created || is_disabled} primary_light onClick={onAction} >
```

Reply:
I think a better approach here would be to use discriminated unions pattern. Instead of having `primary`, `secondary`, `tertiary`, `is_disabled` etc as props in `button` component, we can just have a `type` prop which can accept these values. But that will be out of scope and would require an app wide refactor. Adding onto the technical debt 😢